Model.cs

```diff
@@ -3,18 +3,33 @@
 
 namespace todo
 {
-    public class Todo
+    public class Todo<IDType>
```

Comment:
Can we just store the same type the client wants to send for ID? I may have just used the wrong one.

Reply:
No. CosmosDB uses the id property for the key in its key-value-store and we want both todo and replicache-client-state to be stored in the same collection.

This is pretty ugly. I was considering how to make this cleaner...